lighthouse-core/report/html/templates.html

```diff
@@ -60,13 +60,37 @@
           <span class="lh-audit-group__title"></span>
           <span class="lh-audit-group__itemcount"></span>
           <!-- .lh-audit-group__description will be added here -->
+          <!-- .lh-metrics-toggle will be added here -->
         </div>
         <div class=""></div>
       </div>
     </summary>
   </details>
 </template>
 
+<!-- Lighthouse metrics toggle -->
+<template id="tmpl-lh-metrics-toggle">
+  <div class="lh-metrics-toggle">
+    <div class="lh-metrics-toggle-less lh-metrics-toggle--active">
+      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+        <defs>
+          <path id="a" d="M0 0h24v24H0V0z" />
+        </defs>
+        <clipPath id="b">
```

Comment:
afaik this clippath isn't being used. i guess its just an artifact from sketch exports.